nc/source/ecs/ColliderTree.h

```diff
@@ -15,6 +15,7 @@ namespace nc
     {
         using allow_trivial_destruction = std::true_type;
         using sort_dense_storage_by_address = std::true_type;
+        // ignore callbacks
```

Comment:
Is this a todo?

Reply:
No, just commenting that I've intentionally left out aliases that won't be used.